lib/improver/tests/ensemble_calibration/ensemble_calibration/test_EnsembleCalibration.py

```diff
@@ -333,13 +399,13 @@ def test_wind_speed_realizations_data_check(self):
                  [0.01406566, 0.01406566, 0.01406566]])
         else:
             predictor_data = np.array(
-                [[2.05799912, 9.73470204, 17.41140496],
-                 [25.08810788, 32.7648108, 40.44151372],
-                 [48.11821664, 55.79491955, 63.47162247]])
+                [[2.7344577, 10.298571, 17.862682],
+                 [25.426796, 32.99091, 40.55502],
+                 [48.119133, 55.683247, 63.24736]])
             variance_data = np.array(
-                [[4.26987243, 4.26987243, 4.26987243],
-                 [4.26987243, 4.26987243, 4.26987243],
-                 [4.26987243, 4.26987243, 4.26987243]])
+                [[0.03198197, 0.03198197, 0.03198197],
+                 [0.03198197, 0.03198197, 0.03198197],
+                 [0.03198197, 0.03198197, 0.03198197]])
```

Comment:
With 1000 iterations:
```
8.142177e-10, 8.142177e-10, 8.142177e-10, 8.142177e-10, 8.142177e-10, 8.142177e-10, 8.142177e-10, 8.142177e-10, 8.142177e-10
```

You may have to explain to me why this minimiser makes everything 0 with enough iterations.